examples/rosa-classic-public-with-byo-vpc-byo-iam-byo-oidc/variables.tf

```diff
@@ -1,23 +1,14 @@
 variable "openshift_version" {
-  type    = string
-  default = "4.14.9"
+  type        = string
+  default     = "4.14.9"
+  description = "Desired version of OpenShift for the cluster, for example '4.1.0'. If version is greater than the currently running version, an upgrade will be scheduled."
   validation {
     condition     = can(regex("^[0-9]*[0-9]+.[0-9]*[0-9]+.[0-9]*[0-9]+$", var.openshift_version))
     error_message = "openshift_version must be with structure <major>.<minor>.<patch> (for example 4.13.6)."
   }
 }
 
 variable "cluster_name" {
-  type = string
-}
-
-variable "multi_az" {
-  description = "Create the vpc subnets in only one AZ"
-  type        = bool
-  default     = true
-}
-
-variable "machine_cidr" {
-  type    = string
-  default = "10.0.0.0/16"
+  type        = string
+  description = "Name of the cluster. Cannot exceed 15 characters in length. After the creation of the resource, it is not possible to update the attribute value."
```

Comment:
This length is changing soon, I think we shouldn't explicitly mention the max and let the API

Reply:
Ok, I will remove this from all relevant places

Reply:
removed